packages/sitecore-jss-forms/src/FormField.ts

```diff
@@ -1,15 +1,19 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
 import { HtmlFormField } from './HtmlFormField';
 import { InputViewModel, TitleFieldViewModel, ViewModel, FieldViewModel } from './ViewModel';
 
 export interface FormField<TViewModel extends ViewModel = ViewModel> {
   model: TViewModel;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
@ambrauer In this case if we will add some changes to old packages, we still can break rules, so sometimes better do disable some places, instead of disabling it on entire package. What do you think?

Reply:
@sc-illiakovalenko Agree, better to catch any future uses if we can.